src/upgrade.cpp

```diff
@@ -175,6 +175,16 @@ ConfigObject<ConfigValue>* Upgrade::versionUpgrade(const QString& settingsPath)
             delete oldFile;
         }
         else {
+#elif __WINDOWS__
+        qDebug() << "Config version is empty, trying to read pre-1.12.0 config";
```

Comment:
Oh, in fact not.
The purpose of this code is: "If we can't find a config file at the new location (and consequently config version is empty), then we try to read config from old location before assuming this is a first run"

on winXP, this is dead code because new settings path == old settings path, but it is needed in windows seven+ to ensure we don't loose config.